pkgs/servers/sql/postgresql/ext/vectorchord/package.nix

```diff
@@ -0,0 +1,144 @@
+{
+  buildPgrxExtension,
+  cargo-pgrx_0_14_1,
+  clang,
+  fetchFromGitHub,
+  lib,
+  nix-update-script,
+  postgresql,
+  postgresqlTestExtension,
+  replaceVars,
+  rust-jemalloc-sys,
+  stdenv,
+}:
+let
+  buildPgrxExtension' = buildPgrxExtension.override {
+    # Upstream only works with a fixed version of cargo-pgrx for each release,
+    # so we're pinning it here to avoid future incompatibility.
+    cargo-pgrx = cargo-pgrx_0_14_1;
+  };
+
+  # Follow upstream and use rust-jemalloc-sys on linux aarch64 and x86_64
```

Comment:
This was a problem recently: https://github.com/NixOS/nixpkgs/pull/414234

Reply:
Does anyone have an aarch64-linux machine that can test this to see if it's affected? Otherwise, should I do a similar patch?